RetailCoder.VBE/Inspections/QuickFixes/AggregateInspectionFixAll.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Rubberduck.Inspections.Abstract;
+using System;
+using Rubberduck.VBEditor;
+using System.Collections.Generic;
+
+namespace Rubberduck.Inspections.QuickFixes
+{
+    class AggregateInspectionFixAll: QuickFixBase
+    {
+        List<IInspectionResult> results;
+
+        public AggregateInspectionFixAll(List<IInspectionResult> aggregate) : base(null, new QualifiedSelection(), aggregate[0].Description)
+        {
+            results = aggregate;
+        }
+
+        public override bool CanFixInModule
+        {
+            get
+            {
+                return false;
+            }
+        }
+
+        public override bool CanFixInProject
+        {
+            get
+            {
+                return false;
+            }
+        }
+
+        public override void Fix()
+        {
+            // FIXME fix all the aggregated results with some useful QuickFix ...
```

Comment:
Also since we got no useful Type-Information at this point to select the "correct" quickfix this is really difficult to do